ext/bcmath/tests/bcmod.phpt

```diff
@@ -6,15 +6,274 @@ bcmath
+-16.60 mod 0.15                           = 0
+-16.60 mod -0.01                          = 0
+-16.60 mod 15151324141414.412312232141241 = -16
+-16.60 mod -132132245132134.1515123765412 = -16
+-16.60 mod 141241241241241248267654747412 = -16
+-16.60 mod -149143276547656984948124912   = -16
+-16.60 mod 0.1322135476547459213732911312 = 0
+-16.60 mod -0.123912932193769965476541321 = 0
+
+Number "0.15" (scale 0)
+  0.15 mod 15                             = 0
+  0.15 mod -15                            = 0
+  0.15 mod 1                              = 0
+  0.15 mod -9                             = 0
+  0.15 mod 14.14                          = 0
+  0.15 mod -16.60                         = 0
+  0.15 mod 0.15                           = 0
+  0.15 mod -0.01                          = 0
+  0.15 mod 15151324141414.412312232141241 = 0
+  0.15 mod -132132245132134.1515123765412 = 0
+  0.15 mod 141241241241241248267654747412 = 0
+  0.15 mod -149143276547656984948124912   = 0
+  0.15 mod 0.1322135476547459213732911312 = 0
+  0.15 mod -0.123912932193769965476541321 = 0
+
+Number "-0.01" (scale 0)
+ -0.01 mod 15                             = 0
+ -0.01 mod -15                            = 0
+ -0.01 mod 1                              = 0
+ -0.01 mod -9                             = 0
+ -0.01 mod 14.14                          = 0
+ -0.01 mod -16.60                         = 0
+ -0.01 mod 0.15                           = 0
+ -0.01 mod -0.01                          = 0
+ -0.01 mod 15151324141414.412312232141241 = 0
+ -0.01 mod -132132245132134.1515123765412 = 0
+ -0.01 mod 141241241241241248267654747412 = 0
+ -0.01 mod -149143276547656984948124912   = 0
+ -0.01 mod 0.1322135476547459213732911312 = 0
+ -0.01 mod -0.123912932193769965476541321 = 0
+
+Number "15" (scale 10)
+    15 mod 15                             = 0.0000000000
+    15 mod -15                            = 0.0000000000
+    15 mod 1                              = 0.0000000000
+    15 mod -9                             = 6.0000000000
+    15 mod 14.14                          = 0.8600000000
+    15 mod -16.60                         = 15.0000000000
+    15 mod 0.15                           = 0.0000000000
+    15 mod -0.01                          = 0.0000000000
+    15 mod 15151324141414.412312232141241 = 15.0000000000
+    15 mod -132132245132134.1515123765412 = 15.0000000000
+    15 mod 141241241241241248267654747412 = 15.0000000000
+    15 mod -149143276547656984948124912   = 15.0000000000
+    15 mod 0.1322135476547459213732911312 = 0.0598691150
+    15 mod -0.123912932193769965476541321 = 0.0065352045
+
+Number "-15" (scale 10)
+   -15 mod 15                             = 0.0000000000
+   -15 mod -15                            = 0.0000000000
+   -15 mod 1                              = 0.0000000000
+   -15 mod -9                             = -6.0000000000
+   -15 mod 14.14                          = -0.8600000000
+   -15 mod -16.60                         = -15.0000000000
```

Comment:
Using non integer values for mod is *very* questionable. But I imagine this is the current behaviour.